src/RA_Defs.cpp

```diff
@@ -91,19 +173,77 @@ void RADebugLog(const char* format, ...)
     *p++ = '\n';
     *p = '\0';
 
-    OutputDebugString(NativeStr(buf).c_str());
-
-    //SetCurrentDirectory( g_sHomeDir.c_str() );//?
-    FILE* pf = nullptr;
-    if (fopen_s(&pf, RA_LOG_FILENAME, "a") == 0)
-    {
-        fwrite(buf, sizeof(char), strlen(buf), pf);
-        fclose(pf);
-    }
+    RADebugLogNoFormat(buf);
 }
 
+_Use_decl_annotations_
 BOOL DirectoryExists(const char* sPath)
 {
     DWORD dwAttrib = GetFileAttributes(NativeStr(sPath).c_str());
     return(dwAttrib != INVALID_FILE_ATTRIBUTES && (dwAttrib & FILE_ATTRIBUTE_DIRECTORY));
 }
+
+_Use_decl_annotations_
+ResizeContent::ResizeContent(HWND parentHwnd, HWND contentHwnd, AlignType newAlignType, bool isResize) noexcept
+{
```

Comment:
This constructor can't be constexpr and isn't from a class template so it shouldn't be in the header file.